models/Recording.js

```diff
@@ -58,10 +59,19 @@ function canGetRaw() {
   return false;
 }
 
+function getFileName() {
+  var ext = "";
+  if (this.fileMimeType == 'video/mp4') ext = ".mp4"
+  var dateStr = new Date(this.recordingDateTime)
+    .toLocaleString().replace(" ", "T").replace(/:/g, "-");
```

Comment:
Is the output from `toLocaleString` dependent on the system locale? If so, this isn't going to be all that reliable.

A more robust approach could be to use moment.js or datejs for parsing and formatting the date.